sentry_sdk/integrations/aiohttp.py

```diff
@@ -234,27 +248,80 @@ async def on_request_start(session, trace_config_ctx, params):
 
         trace_config_ctx.span = span
 
+        if params.url.path == "/graphql":
+            trace_config_ctx.request_headers = params.headers
+
+    async def on_request_chunk_sent(session, trace_config_ctx, params):
+        # type: (ClientSession, SimpleNamespace, TraceRequestChunkSentParams) -> None
+        if not hasattr(params, "url") or not hasattr(params, "method"):
+            # these are missing from params in earlier aiohttp versions
+            return
+
+        if params.url.path == "/graphql" and params.method == "POST":
+            with capture_internal_exceptions():
+                trace_config_ctx.request_body = json.loads(params.chunk)
+
     async def on_request_end(session, trace_config_ctx, params):
         # type: (ClientSession, SimpleNamespace, TraceRequestEndParams) -> None
-        if trace_config_ctx.span is None:
```

Comment:
Why no more early return when span is None?

Reply:
We still want to inspect the response for graphql errors regardless of the span being there or not.